analysis/grammar_search_analysis.py

```diff
@@ -242,29 +242,33 @@ def _make_proxy_analysis_results(outdir: str) -> None:
 
 
 def _main() -> None:
-    env_names = [
-        "cover",
-        "blocks",
-        "painting",
-    ]
-    score_function_names = [
-        # "prediction_error",
-        # "hadd_energy_lookaheaddepth0",
-        # "hadd_energy_lookaheaddepth1",
-        # "hadd_energy_lookaheaddepth2",
-        # "exact_energy",
-        # "lmcut_count_lookaheaddepth0",
-        # "hadd_count_lookaheaddepth0",
-        # "exact_count",
-        "expected_nodes",
-    ]
-    run_planning = False
+    args = utils.parse_args(env_required=False,
+                            approach_required=False,
+                            seed_required=False)
+    assert args["excluded_predicates"] == ""  # gets ignored
+    if args["env"] is not None:
+        env_names = [args["env"]]
+    else:
+        del args["env"]
+        env_names = [
```

Comment:
Let's do `env_names = DEFAULT_ENV_NAMES` here and define that constant at top of file?